playbooks/robusta_playbooks/pod_investigator_enricher.py

```diff
@@ -144,6 +153,7 @@ def report_pod_issue(event: KubernetesResourceEvent, pods: List[Pod], issue: Pod
         blocks.append(MarkdownBlock(f"\n\n*{pod_names[0]}* was picked for investigation\n"))
```

Comment:
We need to also handle the case where `additional_blocks` are empty, but we do have the `issue_details`

I think the issue details markdown message should just be: `f"{reason}:{message}"` (instead of "Problem cause" + message)

Reply:
changed